docs/typescript/reference/dbos.md

```diff
@@ -0,0 +1,953 @@
+---
+sidebar_position: 4
+title: DBOS Class
+description: API reference for DBOS
+---
+
+# DBOS Class
+
+The `DBOS` class is the heart of the DBOS Transact programming API, providing decorators to identify key application functions, and accessors for context-dependent state.
```

Comment:
If I understand it correctly, the `DBOS` class contains only static method. So there's no way to instantiate a DBOS object like we do in Python. If so, we might want to clarify that this class only contains static methods.